tests/tests_lite/test_lite.py

```diff
@@ -0,0 +1,10 @@
+from lightning_lite.lite import LightningLite
+
+
+def test_placeholder(tmpdir):
+    assert True
+
+
+@RunIf(min_cuda_gpus=2, standalone=True)
```

Comment:
We can copy the implementation or import it from pl. I leave it up to @carmocca and @Borda to decide what to do here.

Reply:
How would you import it? By adding the file to the pythonpath? Tests are usually not importable.

I vote towards copying. It could be imported later from Lite in PL

Reply:
How about outsourcing this class/function to https://github.com/Lightning-AI/devtools
and install it only for testing...

Reply:
The class should not be outsourced because each argument is library specific

Reply:
I copied over a minimal version of RunIf to make sure the basic setup works (min_cuda and standalone). Once the utilities move, we can then copy over more of the implementation with all the different conditions that apply.